manifests/view.pp

```diff
@@ -9,6 +9,7 @@
     $recursion_match_clients      = 'any',
     $recursion_match_destinations = '',
     $recursion_match_only         = false,
+    $empty_zones                  = '',
```

Comment:
ie., we could be modifying the default behaviour if doing so, depending on bind version and/or whether the view is recursive.